src/libs/ReportUtils.js

```diff
@@ -1229,6 +1203,32 @@ function getMoneyRequestReportName(report, policy = undefined) {
     return payerPaidAmountMesssage;
 }
 
+/**
+ * Get the report given a reportID
+ *
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getReport(reportID) {
+    return lodashGet(allReports, `${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {});
```

Comment:
Why are we returning {} for missing reports? 😬 wouldn't `null` or `undefined` be better in these cases?

Reply:
It is much more intuitive for me to check if a report is missing with `report === null` than `Object.keys(report) === 0`

Reply:
oh, I see that you just moved around this function and it was already like that 😿